develop/tutorials/articles/02-from-liferay-6-to-liferay-7/08-optimizing-existing-plugins-for-liferay-7/06-modularizing-plugins/00-modularizing-plugins-intro.markdown

```diff
@@ -0,0 +1,3 @@
+# Modularizing Plugins [](id=modularizing-plugins)
+
+This is a placeholder.
```

Comment:
Can you write something for this? @jorgeferrer has placed a moratorium on placeholders.